tutorials/dataframe/distrdf002_dask_connection.py

```diff
@@ -0,0 +1,63 @@
+## \file
+## \ingroup tutorial_dataframe
+## \notebook -draw
+## Configure a Dask connection and fill two histograms distributedly.
+##
+## This tutorial shows the ingredients needed to setup the connection to a Dask
+## cluster (e.g. a `LocalCluster` for a single machine). After this initial
+## setup, an RDataFrame with distributed capabilities is created and connected
+## to a Dask `Client` instance. Finally, a couple of histograms are drawn from
+## the created columns in the dataset. Relevant documentation can be found at
+## http://distributed.dask.org/en/stable .
+##
+## \macro_code
+## \macro_image
+##
+## \date February 2022
+## \author Vincenzo Eduardo Padulano
+from dask.distributed import LocalCluster, Client
+import ROOT
+
+# Point RDataFrame calls to Dask RDataFrame object
+RDataFrame = ROOT.RDF.Experimental.Distributed.Dask.RDataFrame
+
+# Setup connection to a Dask cluster. Two ingredients are needed:
+# 1. Creating a cluster object that represents computing resources. This can be
+#    done in various ways depending on the type of resources at disposal. To use
+#    only the local machine (e.g. your laptop), a `LocalCluster` object cam be
+#    used. This step is optional if the cluster resources are setup externally.
+#    More options for cluster creation can be found in the Dask docs
+#    http://distributed.dask.org/en/stable/api.html#cluster .
+# 2. Creating a Dask client object that connects to the cluster. This accepts
+#    directly the object previously created. In case the cluster was setup
+#    externally, you need to provide an endpoint URL to the client, e.g.
+#    'https://myscheduler.domain:8786'.
+
+# In this example, a cluster object is created for the local machine, using
+# multiprocessing (processes=True) on 4 workers (n_workers=4) each using only
+# 1 core (threads_per_worker=1).
+cluster = LocalCluster(n_workers=4, threads_per_worker=1, processes=True)
```

Comment:
can we also have examples, in this tutorial, for condor, slurm, ssh? either commented or uncommented but unused -- so we can point users here to show how it's done, and from here they know what dask docs they have to further read if they want to know more

Reply:
Perhaps commented, since the tutorial needs to be runnable?